src/net.cpp

```diff
@@ -2559,6 +2553,25 @@ bool CConnman::DisconnectNode(const std::string& strNode)
     }
     return false;
 }
+
+bool CConnman::DisconnectNode(const CSubNet& subnet)
```

Comment:
In commit `net: Break disconnecting out of Ban()`,

Why `bool` if the return value is not used?

Reply:
Just to be consistent with the others.